### PR TITLE
Update vcpkg, fix QT paths

### PR DESCRIPTION
**Pull request by nullkiller**
Lets update vcpkg. Fix for SSL, MP3 support, and never FFMPEG

### CODE REVIEW

File: osx/CMakeLists.txt [hunk 1]
```diff
@@ -63,37 +63,11 @@ endif(APPLE)
...
-			execute_process(
-				COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_BINARY_DIR}/bin/\${BUILD_TYPE}/bearer \${CMAKE_INSTALL_PREFIX}/bearer
-				COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_BINARY_DIR}/bin/\${BUILD_TYPE}/iconengines \${CMAKE_INSTALL_PREFIX}/iconengines
-				COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_BINARY_DIR}/bin/\${BUILD_TYPE}/imageformats \${CMAKE_INSTALL_PREFIX}/imageformats
-				COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_BINARY_DIR}/bin/\${BUILD_TYPE}/platforminputcontexts \${CMAKE_INSTALL_PREFIX}/platforminputcontexts
-				COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_BINARY_DIR}/bin/\${BUILD_TYPE}/platforms \${CMAKE_INSTALL_PREFIX}/platforms)
-			")
-		endif()
+		install(CODE "
+			file(WRITE ${CMAKE_INSTALL_PREFIX}/qt.conf
```
Comment (kambala-decapitator) [inline osx/CMakeLists.txt:67]:
```suggestion
			file(WRITE \${CMAKE_INSTALL_PREFIX}/qt.conf
```

**Review by kambala-decapitator — CHANGES_REQUESTED** [review]

File: osx/CMakeLists.txt (hunk 1 above)
Comment (nullkiller) [reply osx/CMakeLists.txt:67]: I copied your code. Or is it windows specific requirement to encode $?

Comment (kambala-decapitator) [reply osx/CMakeLists.txt:67]: you've copied my code incorrectly ;)

`CMAKE_INSTALL_PREFIX` variable is defined only at _install_ time (when cpack runs), but CMakeLists are processed at _configure_ time. Using `\$` inserts literal `$` into the install script.

**Review by kambala-decapitator — APPROVED** [review]
seems good now. Let's also test the produced windows binaries.

File: win/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# We need to keep this code into separate directory so CMake will execute it after all other subdirectories install code
+# Otherwise we can't fix Mac bundle dependencies since binaries wouldn't be there when this code executed
```
Comment (kambala-decapitator) [inline win/CMakeLists.txt:2]: Please fix this comment

**Pull request merged**